mrmustard/math/backend_tensorflow.py

```diff
@@ -370,11 +370,16 @@ def eigvals(tensor: tf.Tensor) -> Tensor:
     def xlogy(x: tf.Tensor, y: tf.Tensor) -> Tensor:
         return tf.math.xlogy(x, y)
 
-    def sqrtm(self, tensor: tf.Tensor, rtol=1e-05, atol=1e-08) -> Tensor:
+    def sqrtm(self, tensor: tf.Tensor, dtype, rtol=1e-05, atol=1e-08) -> Tensor:
```

Comment:
just wondering if the atol and rtol are okay? they seem a bit large (but maybe I'm wrong)

Reply:
Good question.. 
They were set before the backend refactoring, so I never thought too much about it.

If we lower them and the tests still pass, are we "happy"?